v0/destinations/snapchat_conversion/util.js

```diff
@@ -0,0 +1,92 @@
+const get = require("get-value");
+const sha256 = require("sha256");
+const { logger } = require("handlebars");
+
+const { isDefinedAndNotNull } = require("../../util");
+
+function msUnixTimestamp(timestamp) {
+  const time = new Date(timestamp);
+  return time.getTime() * 1000 + time.getMilliseconds();
+}
+
+function getHashedValue(identifier) {
+  if (identifier) {
+    return sha256(identifier);
+  }
+}
+function getNormalizedPhoneNumber(message) {
+  let phoneNumber = message?.context?.traits?.phone?.toString();
```

Comment:
again confirm that no parsing error is happening. add test cases for other usages like this as well and make sure it works properly.

Reply:
Working fine.